src/utils/api.ts

```diff
@@ -0,0 +1,30 @@
+
+export type HTTPMethod = 'GET' | 'POST' | 'PUT' | 'DELETE' | 'PATCH' | 'OPTIONS' | 'HEAD';
```

Comment:
I suggest we establish a consistent naming convention for custom type aliases to improve code readability. This will make it easier to identify types throughout the codebase. Pluralizing it as well gives a sense of several methods, in my opinion:
```suggestion
export type HttpMethods = 'GET' | 'POST' | 'PUT' | 'DELETE' | 'PATCH' | 'OPTIONS' | 'HEAD';
```